Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "adlx"
+name = "adlx-rs"
```

Comment:
Shall we leave the `-rs` suffix out?

Reply:
Yes, leftover from my first take which was bad. Will fix it